data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -318,6 +318,52 @@ then
     record_service_stage_success
 fi
 
+if [ ! -f cmdbootline-nto-bootstrap.done ]; then
+	record_service_stage_start "cmdbootline-nto-bootstrap"
+
+	rm --recursive --force cmdbootline-nto-bootstrap
+
+	# Create folders for overlay and podman command to use them
+	tmpfolder=$(mktemp -d)
+	echo ${tmpfolder}
+	podmancmd="bootkube_podman_run -d --rm "
+	for folder in var/lib/tuned/profile-data etc/modprobe.d etc/sysconfig etc/kubernetes etc/sysctl.d etc/systemd run sys lib/modules var/lib/kubelet usr/lib etc/tuned; do
+		lowerfolder="/${folder}"
+		if [ ! -d "$lowerfolder" ]; then
+			lowerfolder="${tmpfolder}/${folder}/lower"
+			mkdir -p ${lowerfolder}
+		fi
+		upperfolder="${tmpfolder}/${folder}/upper"
+		workfolder="${tmpfolder}/${folder}/work"
+		mkdir -p ${upperfolder}
+		mkdir -p ${workfolder}
+		podmancmd="${podmancmd} -v ${lowerfolder}:/${folder}:O,upperdir=${upperfolder},workdir=${workfolder}"
+	done
+
+	podmancmd="${podmancmd} --name tuned --entrypoint \"cluster-node-tuning-operator\" ${NODE_TUNING_OPERATOR_IMAGE} render-bootcmd-mc --asset-input-dir=/assets/manifests --asset-output-dir=/assets/cmdbootline-nto-bootstrap"
+
+	# need to configure tuned
+	# copy (or create) /etc/tuned/tuned-main.conf to ${tmpfolder}/etc/tuned/upper/tuned-main.conf with "daemon = 0"
+	tunedConfigFileName="${tmpfolder}/etc/tuned/upper/tuned-main.conf"
+	if [ ! -f $tunedConfigFileName ]; then
+		touch $tunedConfigFileName
+		echo "daemon = 0" >$tunedConfigFileName
+	fi
+	sed -i '/^daemon =/{h;s/=.*/= 0/};${x;/^$/{s//daemon = 0/;H};x}' $tunedConfigFileName
+
+	# execute podmancmd
+	eval "CLUSTER_NODE_TUNED_BOOTSTRAP_SAFE_ENV=true $podmancmd"
```

Comment:
It seems the `eval` here does something to `pipefail`, so even if the container run fails we continue and touch `cmdbootline-nto-bootstrap.done`
```suggestion
	CLUSTER_NODE_TUNED_BOOTSTRAP_SAFE_ENV=true $podmancmd
```

Reply:
not sure about the relation between `eval` and `pipefail` problem.
Running locally a script with an intended error does not end with the `flag.file` touched.
```
>-$ 
ls -ltr flag.file
ls: cannot access 'flag.file': No such file or directory

>-$ 
cat xxx.sh 
#!/usr/bin/env bash
set -euoE pipefail ## -E option will cause functions to inherit trap

bootkube_podman_run() {
  # we run all commands in the host-network to prevent IP conflicts with
  # end-user infrastructure.
  dockers run "${@}"
}
podmancmd="bootkube_podman_run  --rm "
podmancmd="${podmancmd} --name tuned --entrypoint \"cluster-node-tuning-operator\" quay.io/jojosneg/origin-cluster-node-tuning-operator:master render -h"

echo "podmancmd: ${podmancmd}"
eval "${podmancmd}"

touch ./flag.file

>-$ 
./xxx.sh 
podmancmd: bootkube_podman_run  --rm  --name tuned --entrypoint "cluster-node-tuning-operator" quay.io/jojosneg/origin-cluster-node-tuning-operator:master render -h
./xxx.sh: line 9: dockers: command not found

>-$ 
ls -ltr flag.file
ls: cannot access 'flag.file': No such file or directory
```

Reply:
Also cannot just delete `eval` as we need the content of the `podmancmd` string to be interpreted to work properly.

Reply:
The problem happens to me when I do not use `eval` to run the command ... 
Anyhow as it is the default entrypoint lets remove it.

Reply:
Let's remove it anyway to make it simpler.